src/emailpro/account/emailpro-account.controller.js

```diff
@@ -1,13 +1,15 @@
-angular.module('Module.emailpro.controllers').controller('EmailProTabAccountsCtrl', ($scope, EmailPro, $q, $stateParams, $translate) => {
+angular.module('Module.emailpro.controllers').controller('EmailProTabAccountsCtrl', ($scope, EmailPro, $q, $stateParams, $translate, WucConverterService) => {
   $scope.stateCreating = EmailPro.stateCreating;
   $scope.stateDeleting = EmailPro.stateDeleting;
   $scope.stateOk = EmailPro.stateOk;
   $scope.stateReopening = EmailPro.stateReopening;
   $scope.stateSuspended = EmailPro.stateSuspended;
   $scope.stateSuspending = EmailPro.stateSuspending;
+  $scope.WucConverterService = WucConverterService;
 
   $scope.stateTaskError = 'TASK_ON_ERROR';
   $scope.stateTaskDoing = 'TASK_ON_DOING';
+  $scope.GB = 'GB';
```

Comment:
This var should be stored in a constant file, (with a more explict name => `SIZE_UNIT` for example ?)

Also you could store `stateTaskError` and `stateTaskDoing` in the file, (I know it's not yours change though)